README

```diff
@@ -1,19 +1,19 @@
 The Steam protocol plugin for bitlbee. This plugin uses the Steam Mobile
 API allowing it to run alongside the main Steam client. It is worth
-noting that the Steam Mobile API is HTTP based, which does lead to mild
+noting that the Steam Mobile API is HTTP based plugin, which does lead to mild
 latency.
 
 ## Package Repositories
 
 There are package repositories available for various distributions.
 
-See: https://jgeboski.github.io
+See: https://jgeboski.github.io  For further details
 
 ## Building and Installing
 
-Make sure bitlbee and its headers have been installed. If bitlbee came
+Make sure, bitlbee and its headers have been installed. If bitlbee came
 from the distribution's repository, it will most likely need the
-development package, usually bitlbee-dev.
+development package which you must install, usually bitlbee-dev.
```

Comment:
This is redundant with the previous part of that clause ("will most likely need" ≈ "you must install").

```suggestion
development package, usually bitlbee-dev.
```